sample/src/main/java/com/friendoye/recyclerxray/sample/MainActivity.kt

```diff
@@ -41,20 +49,27 @@ class MainActivity : AppCompatActivity() {
             isFullDataInAdapter = !isFullDataInAdapter
         }
 
-        RecyclerXRay.settings = XRaySettings.Builder()
-            .withMinDebugViewSize(dip(100))
-            .build()
+        setupRecyclerXRays()
 
         binding.sampleRecyclerView.apply {
             layoutManager = GridLayoutManager(context, 3).apply {
-                spanSizeLookup = SampleAdapterSpanLookup(sampleAdapter)
+                spanSizeLookup = SampleAdapterSpanLookup {
+                    adapter as? ConcatAdapter ?: RecyclerXRay.unwrap(adapter!!)
+                }
             }
             // Test RecyclerXRay
-            adapter = RecyclerXRay.wrap(sampleAdapter)
-            sampleAdapter = RecyclerXRay.unwrap(adapter!!)
+            adapter = ConcatAdapter(
```

Comment:
Add tests for `ConcatAdapter` integration:
1) With wrapping `ConcatAdapter`;
2) With `ConcatAdapter`, inited with wrapped adapters.